Fix #1343: Add mobile SDK license keys into init SDK (#1348)
wultra/enrollment-server: docs/onboarding/Configuration-Verification-Providers.md

The format of the document name is `{country}_{type}_{edition}` according to the response of `/metadata` request.

### Microblink

> The BlinkID component from Microblink is used for document verification. The official documentation can be found at [this link](https://blinkidverify.docs.microblink.com/docs/docver/overview/intro).

Reviewer: 🤔 Sure, it is working but could we figure out something better than _this link_?
```suggestion
The BlinkID component from Microblink is used for document verification. For details, see [the official Microblink documentation](https://blinkidverify.docs.microblink.com/docs/docver/overview/intro).
```

Author: I agree, this is better than `this link`

It is single REST API endpoint `POST /api/v2/docver` protected by Basic Auth. See endpoint specification at [this link](https://blinkidverify.docs.microblink.com/docs/docver/api-docs/api-v2).

When a document is uploaded, then it is temporarily stored in cache on Onboarding server. After all documents are uploaded, the endpoint is called for each document.

The mobile SDK uses Microblink SDK to capture document images. The SDK requires license key, which is stored in Onboarding server configuration per mobile platform.
The license key is sent in response body of `POST /api/identity/document/init-sdk` as `licenseKey` attribute. In request 

> body there should be attribute `mobilePlatform`. Possible values are `IOS` and `ANDROID`.

Author: Removed the end of line in the middle of sentence
